eksrollup/cli.py

```diff
@@ -222,13 +223,15 @@ def update_asgs(asgs, cluster_name):
                 desired_asg_capacity -= 1
                 drain_node(node_name)
                 delete_node(node_name)
-                ca_replicas = check_k8s_autoscaler()
-                logger.info(f'K8s autoscaler deployment is at {ca_replicas} replicas...')
-                if ca_replicas > 0:
-                    logger.info('Need to pause down K8s autoscaler deployment...')
-                    modify_k8s_autoscaler("pause")
-                else:
-                    logger.info('No extra work required for K8s autoscaler deployment...')
+                # ensure nothing reverted scaled-down of autoscaler if enabled:
```

Comment:
This is just fixing the original logic for handling failures - it was trying to scale up CA even if it's disabled for the job.